### ISSUE TITLE
Different behavior on Android and iOS with the delete button

### ISSUE THREAD

**Issue by GerardoTovar** [code-farmz/ng-otp-input#129]
ng-otp-input: 2.0.6
Android: 13
Angular: 18.2.13
Ionic: 8.4.1

On Android, when you delete, it jumps to the next box and if it is empty, it does not go back. On iOS, it works perfectly.

### **Android:**

https://github.com/user-attachments/assets/93056e39-e0e9-402d-b886-db26571088b0

### **Ios:**

https://github.com/user-attachments/assets/746689a0-faf8-452d-b337-653331425b42

Honestly, I don't know if it's considered a problem. I think it was in 1.9.3 and on Android when you pressed the delete button it would delete its content and that's it. And if it was empty and you pressed the back button it would do nothing. However, in this version it jumps to the next box. On iOS it works the same and perfectly from 1.9.3 to 2.0.6.

**Note: Thanks a lot for the 2.0.6 fix, that was quick.**

**Comment by code-farmz** [comment]
Hi @GerardoTovar 

Thanks for posting this issue tried this with couple of android devices and wasn't able to reproduce. It might be device specific or android version specific issue.I will monitor this and see if I can reproduce this and fix.

https://github.com/user-attachments/assets/a71e46d4-11d2-4f0f-b6af-27f3314d6d86

**Comment by GerardoTovar** [comment]
You are absolutely right, I tried it on other devices and had no problem. These devices used the Gboard app. But the one that had problems had a custom rom with the keyboard "Android Keyboard (AOSP)", I downloaded the Gboard keyboard and the error disappeared. The error only occurs with this keyboard.

Of course very few people will have a different keyboard. But it's good to know that this happens.

![3d762124-350c-4f29-82dc-7aafaf42f287](https://github.com/user-attachments/assets/89ba6255-ea90-492c-82fe-4a80177ec66d)
